nion/swift/ImageCanvasItem.py

```diff
@@ -240,13 +240,18 @@ class ScaleMarkerCanvasItemComposer(CanvasItem.BaseComposer):
     def __init__(self, canvas_item: CanvasItem.AbstractCanvasItem, layout_sizing: CanvasItem.Sizing,
                  cache: CanvasItem.ComposerCache, dimensional_calibration: typing.Optional[Calibration.Calibration],
                  info_text: str, screen_pixel_per_image_pixel: typing.Optional[float],
-                 get_font_metrics_fn: typing.Callable[[str, str], UISettings.FontMetrics]) -> None:
+                 get_font_metrics_fn: typing.Callable[[str, str], UISettings.FontMetrics],
+                 draw_on_right: bool = False,
+                 background: bool = False,
+                 include_info_text: bool = True) -> None:
         super().__init__(canvas_item, layout_sizing, cache)
         self.__dimensional_calibration = dimensional_calibration
         self.__info_text = info_text
         self.__screen_pixel_per_image_pixel = screen_pixel_per_image_pixel
         self.__get_font_metrics_fn = get_font_metrics_fn
-
```

Comment:
Can you restore blank line before the next method definition? There are several minor formatting issues with this PR that should be addressed before merging.